core/src/main/java/org/apache/accumulo/core/util/compaction/CompactionServicesConfig.java

```diff
@@ -53,6 +53,7 @@ private long getDefaultThroughput(AccumuloConfiguration aconf) {
       return aconf.getAsBytes(Property.TSERV_MAJC_THROUGHPUT);
     }
 
+    // TODO is this supposed to be getting the default value or should it be reading from config?
```

Comment:
Self reminder to remove this TODO or fix if needed

Reply:
Can remove the TODO.  This code will use a deprecated property if its set as the default otherwise if falls back to the default value for the newer property.  This default is only used when  per compaction service config does not set a rate.